pkgs/by-name/en/ente-desktop/package.nix

```diff
@@ -63,7 +63,10 @@ stdenv.mkDerivation (finalAttrs: {
 
   # Path to vips (otherwise it looks within the electron derivation)
   postPatch = ''
-    substituteInPlace src/main/services/image.ts --replace-fail "process.resourcesPath" "\"$out/share/ente-desktop/resources\""
+    for file in src/main/services/image.ts src/main.ts
+    do
+      substituteInPlace $file --replace-fail "process.resourcesPath" "\"$out/share/ente-desktop/resources\""
+    done
```

Comment:
```suggestion
    substituteInPlace src/main/services/image.ts src/main.ts \
      --replace-fail "process.resourcesPath" '"$out/share/ente-desktop/resources"'
```
that works, too

Reply:
Actually that does not work, `$out` is not expanded anymore because you added single quotes and now no resource will be found because `$out` will be included literally. See https://github.com/NixOS/nixpkgs/pull/410214